subprojects/docs/src/samples/signing/in-memory/groovy/gradle.properties

```diff
@@ -0,0 +1,61 @@
+# Typically set in ~/.gradle/gradle.properties or passed via environment variables instead
+signingPassword = gradle
+signingKey = -----BEGIN PGP PRIVATE KEY BLOCK-----\n\
```

Comment:
❌ I'd probably run the example using environment variables and not tell people to put their private keys in clear-text in gradle.properties. That's very unsafe.

The syntax would be `ORG_GRADLE_PROJECT_signingKey=...`

Reply:
I already explained how to use env variables to pass these in the user manual chapter but you're right, I should remove these from here as well so nobody gets the wrong idea.

Done in 5e3537d.